foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchConfig.java

```diff
@@ -35,6 +35,7 @@
     @NotNull
     @JsonProperty
     private String cluster;
+    private String tableNamePrefix = "foxtrot";
```

Comment:
Ok

----
Upset the established order, and everything becomes chaos. I'm an agent of
chaos. Oh and you know the thing about chaos, it's fair.

On Wed, Dec 9, 2015 at 3:35 PM, Swapnil Marghade <notifications@github.com>
wrote:

> In
> foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchConfig.java
> <https://github.com/Flipkart/foxtrot/pull/97#discussion_r47071202>:
>
> > @@ -35,6 +35,7 @@
> >      @NotNull
> >      @JsonProperty
> >      private String cluster;
> > +    private String tableNamePrefix = "foxtrot";
>
> existing code which creates index prefix looks like this
>
> return String.format("%s-%s-%s-", ElasticsearchUtils.TABLENAME_PREFIX,
> table, ElasticsearchUtils.TABLENAME_POSTFIX);
>
> Followed similar convention and named it as table name prefix.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Flipkart/foxtrot/pull/97/files#r47071202>.
>